sample/index.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
 // This is for testing the overall integration
-import * as ApplicationInsights from "applicationinsights";
+import ApplicationInsights = require("applicationinsights");

```

Comment:
It relates to them using `export = ApplicationInsights` rather than `export ApplicationInsights` [here](https://github.com/Microsoft/ApplicationInsights-node.js/blob/develop/applicationinsights.ts#L287)  The relevant typescript documentation is [here](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require). I think this was a recent change in TypeScript to require this import form for the corresponding export form, but I remember needing to make this change to the subs too.